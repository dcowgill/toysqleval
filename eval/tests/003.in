create table test (time timestamp);
insert into test values ('1935-12-30 20:50:47 EST');
insert into test values ('Sun May 10 21:39:19 -0400 2009');
insert into test values ('1927-11-05T05:38:35.346211401-05:00');
insert into test values ('Thu, 30 Jun 1960 21:24:07 -0400');
insert into test values ('Fri Oct 24 12:49:33 EST 1924');
insert into test values ('Fri, 19 Sep 1958 20:18:11 EDT');
insert into test values ('1890-04-14T17:37:15-05:00');
select * from test;
select time||time from test;
