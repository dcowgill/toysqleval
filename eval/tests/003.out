OK
OK
OK
OK
OK
OK
OK
OK
 time                     
---------------------------
 1935-12-30T20:50:47-05:00
 2009-05-10T21:39:19-04:00
 1927-11-05T05:38:35-05:00
 1960-06-30T21:24:07-04:00
 1924-10-24T12:49:33-05:00
 1958-09-19T20:18:11-04:00
 1890-04-14T17:37:15-05:00
 ?                                                   
------------------------------------------------------
 "1935-12-30T20:50:47-05:001935-12-30T20:50:47-05:00"
 "2009-05-10T21:39:19-04:002009-05-10T21:39:19-04:00"
 "1927-11-05T05:38:35-05:001927-11-05T05:38:35-05:00"
 "1960-06-30T21:24:07-04:001960-06-30T21:24:07-04:00"
 "1924-10-24T12:49:33-05:001924-10-24T12:49:33-05:00"
 "1958-09-19T20:18:11-04:001958-09-19T20:18:11-04:00"
 "1890-04-14T17:37:15-05:001890-04-14T17:37:15-05:00"
